### ISSUE TITLE
keyboard access (Ctrl + Enter) to edit a mermaid diagram does not work

### ISSUE THREAD

**Issue by jngbng** [outline/outline#12865]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### This is not related to configuring Outline

- [x] The issue is not related to self-hosting config

### Current Behavior

https://github.com/outline/outline/pull/11834  introduced keyboard access to enter mermaid edit mode.

However, it seems not work on windows.

### Expected Behavior

When a mermaid diagram is focused (diagram is decorated with a blue stoke line and a small floating menu box is shown), hitting `ctrl + enter` should change the diagram to editing mode. (on Windows OS)

### Steps To Reproduce

1. On Windows, using Chrome or Edge browser, access an outline page which includes a mermaid diagram.
2. navigate to mermaid diagram using keyboard (arrow down).
3. See the mermaid diagram is focused.
4. Hit `ctrl + enter`.
5. Check if the selected mermaid diagram changes to editing mode.

### Environment

```markdown
- Outline: v1.8.1
- Browser: Chrome Version 149.0.7827.103 (Official Build) (64-bit) / Windows
```

**Comment by tommoor** [comment]
Closed by https://github.com/outline/outline/pull/12867
